Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2082,6 +2088,11 @@ - (void)didTapVideoViewItem:(ConversationViewItem *)viewItem
     OWSAssert(attachmentStream);
 
     [self dismissKeyBoard];
+    // In case we were presenting edit menu, we need to become first responder before presenting another VC
+    // else UIKit won't restore first responder status to us when the presented VC is dismissed.
+    if (!self.isFirstResponder) {
+        [self becomeFirstResponder];
+    }
```

Comment:
This fixes one repro (tapping media after showing menu controller) but not others (e.g. showing an alert (e.g. by tapping "a call item") after showing menu controller.  

I've got a counter-proposal here that covers all of the repros I know of and IMO feels less whack-a-mole:

https://github.com/signalapp/Signal-iOS/pull/3516

Reply:
Much better! I'll close this.